src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -32549,7 +33072,14 @@ static void M1<T>(T t1)
             comp.VerifyDiagnostics();
 
             var comp2 = CreateNullableCompilation(new[] { source, lib_cs, NotNullAttributeDefinition });
-            comp2.VerifyDiagnostics();
+            comp2.VerifyDiagnostics(
+                // (4,32): warning CS8607: A possible null value may not be assigned to a target marked with the [DisallowNull] attribute or output to a target marked with the [NotNull] attribute
+                //     [NotNull]public TOpen P1 { get; set; } = default!;
+                Diagnostic(ErrorCode.WRN_DisallowNullAttributeForbidsMaybeNullAssignment, "get;").WithLocation(4, 32),
+                // (14,35): warning CS8607: A possible null value may not be assigned to a target marked with the [DisallowNull] attribute or output to a target marked with the [NotNull] attribute
+                //     [NotNull]public TStruct? P5 { get; set; }
+                Diagnostic(ErrorCode.WRN_DisallowNullAttributeForbidsMaybeNullAssignment, "get;").WithLocation(14, 35)
```

Comment:
💡 It would be good to have a test showing that the value can be assigned in a constructor to eliminate the warning (may already exist as a test).

Reply:
❔ I didn't see a test that used the following in a constructor:

```
P5 = new TStruct();
```

Unlike the newly-added `NotNull_Property_InitializedInConstructor` cases, I would expect the above to not produce a warning _and_ resolve the warning reported on the property definition in the two tests here.